packages/core/src/stream-loading/__tests__/stream-loader.test.ts

```diff
@@ -65,8 +65,9 @@ describe('StreamLoader querying against real Ceramic node', () => {
 
     dispatcherIpfs = await createIPFS()
     dispatcher = await createDispatcher(dispatcherIpfs, TOPIC)
-    // speed up how quickly the dispatcher gives up on loading a non-existent commit from ipfs.
-    dispatcher._ipfsTimeout = 100
+    // We need this timeout to be short enough that the non-existent lookup timesout before
```

Comment:
it's also just to make the tests run faster.  Any reason for updating this comment?

Reply:
Because its not just about making the test faster. If you remove that line the test fails because the ipfsTimeout by default is longer that the test timeout. I changed the comment to make that clear.